composer.json

```diff
@@ -24,6 +24,6 @@
         "symfony/security-csrf": "~2.5",
         "symfony/twig-bridge":   "~2.5",
         "org_heigl/daterange": "~1.1",
-        "guzzlehttp/guzzle": "^6.1"
+        "guzzlehttp/guzzle": "~6.2.1"
```

Comment:
I released that the addition of the guzzle dependency was recent (as I had to update my fork to get it). As all the other dependencies used the '~' operator rather than '^', I assumed that tighter version constraints was a project preference.

What version constraint should I change it to?